src/main/java/transforms/CombineDateTimeTransform.java

```diff
@@ -79,22 +77,24 @@ public Object apply( Map<String, String> row ) {
             return null;
         }
         final JavaDateTimeHelper tHelper = new JavaDateTimeHelper( this.timezone,
-                timePattern );
+                this.timePattern, this.shouldAddTimezone );
 
         LocalTime time;
         try {
             time = tHelper.parseTime( t );
         } catch ( Exception e ) {
-            time = dHelper.parseDateTimeAsTime( d );
+            time = tHelper.parseDateTimeAsTime( t );
         }
 
-        if ((date == null) || (time == null)){return null;};
+        if ( ( date == null ) || ( time == null ) ) {return null;}
```

Comment:
can you remove the `return null;` onto a separate line? Also the inner parenthesis are unnecessary and the code is a bit cleaner if they're removed:
```
if (date == null || time == null) {
  return null;
}